processor/spanmetricsprocessor/config.go

```diff
@@ -86,6 +86,39 @@ type Config struct {
 	// LaunchDarklyKey defines the LaunchDarkly key.
 	// Optional. Only required when `EnableFeatureFlag` is `true`.
 	LaunchDarklyKey string `mapstructure:"launch_darkly_key"`
+
+	// Transforms takes in a list of Actions to perform on metrics generated by span metrics processor.
+	// currently there is only one action, the rename action. This allows a user to specify metrics with certain attributes
+	// to have a specific metric name. By default, the generated metric names for Rate / Error is "calls_total" and Duration is
+	// "latency".
+	Transforms []Transform `mapstructure:"transforms"`
+}
+
+// Transform defines the new metric names and the attributes to match on in order for the rename to occur
```

Comment:
I'm confused about the renaming behaviour.

I think that should be a `Rule` type as `matchAny`, `matchAll`, `matchNone` to cover the cases. But we can start with what we need. Looks like we want a `matchAll` rule for our requirements?